app/assets/javascript/app/assets/styles/components/Viewer.scss

```diff
@@ -46,13 +46,19 @@
 
       padding-top: ($elem_header-height - $font_size) / 6;
 
-      a {   
+      a, button {   
         display: inline-block;
 
         color: $color_font-light; 
         font-size: $font_size;
 
-        & + a { margin-left: $font_size * 2; }
+        & + a, & + button { margin-left: $font_size * 2; }
+      }
+
+      button {
+        background: none; 
+        border: none;
+        text-decoration: underline;
```

Comment:
I think it's a little less maintainable to have button stylings spread everywhere, but that's a topic for a future project.